src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/controller/RecallController.kt

```diff
@@ -57,4 +57,19 @@ class RecallController(private val recallService: RecallService) {
     ],
   )
   fun getRecall(@PathVariable recallUuid: UUID): Recall = recallService.findRecallByUuid(recallUuid)
+
+  @GetMapping("/person/{prisonerId}")
+  @PreAuthorize("hasAnyRole('ROLE_REMAND_AND_SENTENCING', 'ROLE_RELEASE_DATES_CALCULATOR',  'ROLE_REMAND_SENTENCING__RECORD_RECALL_RW')")
```

Comment:
Are these roles correct? I'd expect it to just be `ROLE_REMAND_SENTENCING__RECORD_RECALL_RO` and be creating a new read only role.

Reply:
yes I think you're right - but I'd like to defer that please and discuss with @carloveo-moj and @joelstobart-moj 

will have to modify the system client to add the correct roles and then will come bck and refactor these endpoints